app/util/analytics.py

```diff
@@ -194,8 +184,8 @@ def send_request(self):
                    "tool_ver": self.analytics.tool_version,
                    "exp_dur": self.analytics.duration,
                    "act_dur":  self.analytics.actual_duration,
-                   "sel_count": self.analytics.selenium_test_count,
-                   "jm_count": self.analytics.jmeter_test_count,
+                   "full_selenium": self.analytics.selenium_test_count,
```

Comment:
suggest changing to `selenium_test_rates` or `selenium_test_run_rate` or `rate_by_selenium_test_name`